src/functions/meetup.js

```diff
@@ -260,7 +261,6 @@ exports.handler = async (event, context, callback) => {
       console.log(
         `Not prod so not creating contentful event for ${meetup.eventName}`
       )
-      return
```

Comment:
is this not needed?

Reply:
Not really, we're not using the return value from this iteration